3rdparty/openjpeg/openjp2/j2k.c

```diff
@@ -6447,7 +6447,7 @@ OPJ_BOOL opj_j2k_set_threads(opj_j2k_t *j2k, OPJ_UINT32 num_threads)
     return OPJ_FALSE;
 }
 
-static int opj_j2k_get_default_thread_count()
+static int opj_j2k_get_default_thread_count(void)
```

Comment:
Please do NOT modify 3rdparty sources.
Silence related warnings through `ocv_warnings_disable()` instead.